rules/windows/defense_evasion_process_termination_followed_by_deletion.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/11/04"
+maturity = "production"
+updated_date = "2020/11/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process termination followed by the deletion of its executable file. Malware, tools, or other non-native
+files dropped or created on a system by an adversary may leave traces to indicate to what was done within a network and
+how. Removal of these files can occur during an intrusion, or as part of a post-intrusion process to minimize the
+adversary's footprint.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Process Termination Followed by Deletion"
+risk_score = 43
+rule_id = "09443c92-46b3-45a4-8f25-383b028b258d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=5s
```

Comment:
do you anticipate noise for this rule, or is the 5s timeframe enough to cut down FP's?

Reply:
5s should reduce noise as well as excluding signed stuff